apps/client/src/assets/i18n/de-DE.json

```diff
@@ -501,6 +501,7 @@
   "SIMULATOR": {
     "Fixed_notification_number": "Konstanter Klangeffekt",
     "Starting_echo_number": "Klangeffekt #",
+    "Break_on_reclaim": "Split macros on Reclaim point-of-no-return",
```

Comment:
You don't have to put english names as placeholders, the translation library uses english as fallback no matter what :) You can probably leave these here for now tho, I'm curious how the translation tool will handle this (crowdin).